### PR TITLE
Veles from Codilime

### PR DESCRIPTION
**Pull request by FakeApate**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/193715)

### CODE REVIEW

**Comment by FakeApate** [comment]
@microsoft-github-policy-service agree

File: manifests/c/Codilime/Veles/2018.05.0.TIF/Codilime.Veles.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Codilime.Veles
+PackageVersion: 2018.05.0.TIF
```
Comment (Trenly) [inline manifests/c/Codilime/Veles/2018.05.0.TIF/Codilime.Veles.installer.yaml:5]:
```suggestion
PackageVersion: 2018.05
```
The PackageVersion should match what is written to control panel, or `AppsAndFeaturesEntries` need to be specified. After changing this value, the version folder the manifest is in will need to be renamed

File: manifests/c/Codilime/Veles/2018.05.0.TIF/Codilime.Veles.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Codilime.Veles
+PackageVersion: 2018.05.0.TIF
```
Comment (Trenly) [inline manifests/c/Codilime/Veles/2018.05.0.TIF/Codilime.Veles.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2018.05
```
The PackageVersion should match what is written to control panel, or `AppsAndFeaturesEntries` need to be specified. After changing this value, the version folder the manifest is in will need to be renamed

File: manifests/c/Codilime/Veles/2018.05.0.TIF/Codilime.Veles.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: Codilime.Veles
+PackageVersion: 2018.05.0.TIF
```
Comment (Trenly) [inline manifests/c/Codilime/Veles/2018.05.0.TIF/Codilime.Veles.yaml:5]:
```suggestion
PackageVersion: 2018.05
```
The PackageVersion should match what is written to control panel, or `AppsAndFeaturesEntries` need to be specified. After changing this value, the version folder the manifest is in will need to be renamed

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Verify:
image
image

**Review by stephengillie — APPROVED** [review]

**Comment by FakeApate** [comment]
@microsoft-github-policy-service agree

**Comment by Trenly** [comment]
[Policy] Reset Feedback

**Review by stephengillie — APPROVED** [review]

**Pull request merged**